src/features/compensations/components/AllCompensations/components/ToolTipTable/ToolTipTable.tsx

```diff
@@ -1,10 +1,9 @@
-import { observer } from 'mobx-react-lite';
 import { formatMoney } from '../../../../../../common/utils/formatMoney';
 
-function ToolTipTable({
+export function ToolTipTable({
   compensations,
 }: {
-  compensations: EmployeeCompensationsAllItemType[]
+  compensations: EmployeeAllCompensationsItemType[]
 }) {
   return (
     <div className="compensations-tooltip-item">
```

Comment:
this class is not properly named according to the BEM methodology

Link to [BEM documentation](https://en.bem.info/methodology/naming-convention/#%D1%81%D1%82%D0%B8%D0%BB%D1%8C-two-dashes)

please check that all files have the correct class names.